backend/src/index.ts

```diff
@@ -176,6 +177,7 @@ class Server {
 
       await opEnergyIndex.runMainUpdateLoop();
 
+      await opBlockHeaderService.$syncOlderBlockHeader();
```

Comment:
1. opEnergyIndex has runMainUpdateLoop() already: so this line should be moved there;
2. but I think, that it is better to call $syncOlderBlockHeader() from 2 functions:
1. setUpHttpApiRoutes at https://github.com/dambaev/op-energy/blob/a8bbda9d2629034731830108de62450b7de9ac96/backend/src/oe/index.ts#L14 which will be called before entering the main loop
2. handleNewBlock at https://github.com/dambaev/op-energy/blob/a8bbda9d2629034731830108de62450b7de9ac96/backend/src/oe/index.ts#L32 which will be called each time new block will be discovered

this way bitcoin node will not be unnecessary questioned with current tip request as well as DB query....
even more: handleNewBlock() is already containing block header of discovered block, which means, that it can be used as current tip instead of performing new api call to bitcoin node

Reply:
Make sense updated the code